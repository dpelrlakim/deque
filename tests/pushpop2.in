newInit a
pop_back a
pop_back a
pop_back a
pop_back a
pop_back a
pop_back a
pop_back a
push_front a -7
pop_back a
push_front a -8
pop_back a
push_front a -9
at a 0
pop_back a
push_front a 1
pop_back a
push_front a 2
pop_back a
push_front a 3
pop_back a
pop_back a
push_front a -10
pop_back a
newInit a
push_front a -6
push_front a -5
push_front a -4
push_front a -3
push_front a -2
push_front a -1
push_front a 0
[] a 0
[] a 9
[] a 14
