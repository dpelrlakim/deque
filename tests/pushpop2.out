a: 2 3 5 7 11 13 17 19
a: 2 3 5 7 11 13 17
a: 2 3 5 7 11 13
a: 2 3 5 7 11
a: 2 3 5 7
a: 2 3 5
a: 2 3
a: 2
a: -7 2
a: -7
a: -8 -7
a: -8
a: -9 -8
-9
a: -9 -8
a: -9
a: 1 -9
a: 1
a: 2 1
a: 2
a: 3 2
a: 3
a:
a: -10
a:
a: 2 3 5 7 11 13 17 19
a: -6 2 3 5 7 11 13 17 19
a: -5 -6 2 3 5 7 11 13 17 19
a: -4 -5 -6 2 3 5 7 11 13 17 19
a: -3 -4 -5 -6 2 3 5 7 11 13 17 19
a: -2 -3 -4 -5 -6 2 3 5 7 11 13 17 19
a: -1 -2 -3 -4 -5 -6 2 3 5 7 11 13 17 19
a: 0 -1 -2 -3 -4 -5 -6 2 3 5 7 11 13 17 19
0
a: 0 -1 -2 -3 -4 -5 -6 2 3 5 7 11 13 17 19
5
a: 0 -1 -2 -3 -4 -5 -6 2 3 5 7 11 13 17 19
19
a: 0 -1 -2 -3 -4 -5 -6 2 3 5 7 11 13 17 19
